open_spiel/game_transforms/repeated_game_test.cc

```diff
@@ -124,6 +124,8 @@ void RepeatedPrisonersDilemaTest() {
   SPIEL_CHECK_EQ(repeated_game->GetType().min_num_players, 2);
   SPIEL_CHECK_EQ(repeated_game->GetType().utility,
                  GameType::Utility::kGeneralSum);
+  // repeated_game->UtilitySum() should raise an error. This is checked in 
+  // game_transforms_test.py as it's simpler to catch the error from Python.
```

Comment:
I based this off of https://github.com/deepmind/open_spiel/blob/2acbd485af75fc92a4815fefd610c37292537127/open_spiel/tests/basic_tests.cc#L224